frontend/src/interfaces/canva-interfaces.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react';
+
+export interface MenuItem {
+  action: () => void;
+  text: string;
+  icon: ReactNode;
+  name: string;
```

Comment:
I guess that `names` is a constricted set of predefined names, right? If so, we can change it into a static type like this `type MenuItemName = 'change-color' | 'change-size' | 'clear-canvas' | ...` Again, it's not necessary, but this way we can clearly see that this is not a regular string